lib/stm32/f4/rcc.c

```diff
@@ -743,6 +727,8 @@ void rcc_clock_setup_hse_3v3(const struct rcc_clock_scale *clock)
 	rcc_wait_for_osc_ready(RCC_PLL);
 
 	/* Configure flash settings. */
+	flash_dcache_enable();
+	flash_icache_enable();
```

Comment:
up to you.  You could also add bools to the struct.  You could also simply document that it's always turned on now, and see if anyone actually complains :)